Sources/SwiftFormat/PrettyPrint/TokenStreamCreator.swift

```diff
@@ -1453,6 +1453,8 @@ fileprivate final class TokenStreamCreator: SyntaxVisitor {
   }
 
   override func visit(_ node: IfConfigDeclSyntax) -> SyntaxVisitorContinueKind {
+    // there has to be a break after an #endif
+    after(node.poundEndif, tokens: .break(.same, size: 0))
```

Comment:
This is the number of blank spaces that would be inserted, if this break was interpreted as a space instead of a newline. To be honest it feels like this should be a non-discretionary line break, but this is how it's done in a very similar case `PostfixIfConfigExpr`, so I am following that.